src/main/java/in/org/projecteka/hdaf/link/HIPClient.java

```diff
@@ -18,26 +22,35 @@ public HIPClient(WebClient.Builder webClientBuilder) {
         this.webClientBuilder = webClientBuilder;
     }
 
-    public Mono<PatientLinkReferenceResponse> linkPatientCareContext(String patientId, PatientLinkReferenceRequest patientLinkReferenceRequest, String url) {
+    public Mono<PatientLinkReferenceResponse> linkPatientCareContext(
+            String patientId, PatientLinkReferenceRequest patientLinkReferenceRequest, String url) {
         Patient patientInHIP = toHIPPatient(patientId, patientLinkReferenceRequest);
 
-        in.org.projecteka.hdaf.link.link.model.hip.PatientLinkReferenceRequest patientLinkReferenceRequestHIP = new in.org.projecteka.hdaf.link.link.model.hip.PatientLinkReferenceRequest(
-                patientLinkReferenceRequest.getTransactionId(), patientInHIP);
+        in.org.projecteka.hdaf.link.link.model.hip.PatientLinkReferenceRequest patientLinkReferenceRequestHIP =
+                new in.org.projecteka.hdaf.link.link.model.hip.PatientLinkReferenceRequest(
+                        patientLinkReferenceRequest.getTransactionId(), patientInHIP);
```

Comment:
Can we expect `in.org.projecteka.hdaf.link.link.model.hip.PatientLinkReferenceRequest` this to be passed from consumer of HIPClient?